app/main/views/metrics.py

```diff
@@ -35,6 +36,8 @@ def get_metrics():
             metrics['brief_response_count'] = {"value": kv['data']['total'], "ts": updated_at}
         elif kv['key'] == 'supplier_metrics':
             metrics['supplier_count'] = {"value": kv['data']['total'], "ts": updated_at}
+        elif kv['key'] == 'metrics_as_at':
+            metrics['month_ending'] = {"value": kv['data']['monthEnding'], "ts": updated_at}
```

Comment:
Did you consider using the `insight_service` to get the value of `thisMonth` for the most recent month?